main/tests/server/src/com/google/refine/commands/column/AddColumnCommandTests.java

```diff
@@ -0,0 +1,56 @@
+
+package com.google.refine.commands.column;
+
+import static org.mockito.Mockito.*;
+import static org.testng.Assert.*;
```

Comment:
In general, it's best to avoid importing everything in a package like this. For example, it seems like `when` is the only item we need from mockito here, so we could replace this import statement with `import static org.mockito.Mockito.when;`.